### PR TITLE
fix ec cocos

### PR DESCRIPTION
**Pull request by jmcclena**
@smithsp @orso82 With this fix, the toray regression script should pass for the `all_omas` OMFIT branch  https://github.com/gafusion/OMFIT-source/pull/6454

### CODE REVIEW

File: omas/omas_cocos.py [hunk 1]
```diff
@@ -338,8 +338,8 @@
 _cocos_signals['divertors.divertor.:.power_currents.data']='?'                                           #[ADD?]# 1.750000 # current  [W]
 
 # EC_LAUNCHERS
-_cocos_signals['ec_launchers.beam.:.launching_position.phi']='?'                 #[ADD?]# 2.000000 # phi  [rad]
-_cocos_signals['ec_launchers.beam.:.steering_angle_pol']='?'                     #[ADD?]# 2.000000 # _pol  [rad]
+_cocos_signals['ec_launchers.beam.:.launching_position.phi']='TOR'                      # 2.000000 # phi  [rad]
+_cocos_signals['ec_launchers.beam.:.steering_angle_pol']=None                           # 2.000000 # _pol  [rad]
```
Comment (smithsp) [inline omas/omas_cocos.py:342]: I find this odd to have no COCOS dependence.

**Review by smithsp — COMMENTED** [review]
@bclyons12 Can you check the COCCOS here?

File: omas/omas_cocos.py (hunk 1 above)
Comment (jmcclena) [reply omas/omas_cocos.py:342]: hmm... yeah. Maybe it should be 'POL' now that you mention it.

Comment (orso82) [reply omas/omas_cocos.py:342]: @jmcclena the definition is: "Steering angle of the EC beam in the R,Z plane (from the -R axis towards the -Z axis), angle_pol=atan2(-k_Z,-k_R), where k_Z and k_R are the Z and R components of the mean wave vector in the EC beam". The COCOS definition for the poloidal angle is starting from the +R axis (and COCOS 11 has the polodal angle to be clockwise). This means that the `steering_angle_pol` definition in IMAS is not consistent with COCOS 11. I would just leave it as `None`.

Comment (jmcclena) [reply omas/omas_cocos.py:342]: So should we merge this then?

**Comment by smithsp** [comment]
Actually, here is the error
```
Run actions/setup-python@v2
  with:
    python-version: 3.6
    token: ***
Version 3.6 was not found in the local cache
Error: Version 3.6 with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```
Maybe a fluke, as `3.6.15` and others are in the link.

**Comment by orso82** [comment]
The regression failing is strange. The issue seems to be with the GitHub CI and I would be ok to merge as is

**Comment by smithsp** [comment]
I see that is an option.  I am OK as well.  I commented on https://github.com/actions/setup-python/issues/585 to try to track GitHub CI problem.

**Pull request merged**